docs/pages/includes/config-reference/instance-wide.yaml

```
@@ -55,10 +55,18 @@ teleport:
         # token_name: /var/lib/teleport/tokenjoin
         token_name: "token-name"
 
-    # Optional CA pin of the auth server. This enables a more secure way of
-    # adding new nodes to a cluster. See "Adding Nodes to your Cluster"
-    # (https://goteleport.com/docs/setup/admin/adding-nodes).
-    ca_pin: "sha256:7e12c17c20d9cb504bbcb3f0236be3f446861f1396dcbb44425fe28ec1c108f1"
+    # Optional CA pin of the Auth Service, which enables new agents to trust a
+    # Teleport cluster when joining via the Auth Service directly. This can be
+    # either the literal value of the CA pin or an absolute path to a file
+    # consisting only of the CA pin.
+    #
+    # You can also specify the value of ca_pin as a list of CA pins or file
```

```suggestion
    # You can also specify the value of the ca_pin key as a comma-separated list of CA pin values or file paths.
```

This might be wrong--I was guessing comma-separated, but maybe it's one value per line or spaces?

---

It's a custom type that Teleport can [unmarshal](https://github.com/gravitational/teleport/blob/master/api/utils/strings.go#L48-L68) either as a single string or Go slice of strings. I'll add an example to make this clearer!